pom.xml

```
@@ -620,13 +621,30 @@
             <artifactId>revapi-java</artifactId>
             <version>${revapi-java.version}</version>
           </dependency>
+          <dependency>
+            <groupId>org.revapi</groupId>
+            <artifactId>revapi-reporter-json</artifactId>
+            <version>${revapi-reporter-json-version}</version>
+          </dependency>
         </dependencies>
         <configuration>
+          <failBuildOnProblemsFound>false</failBuildOnProblemsFound>
+          <checkDependencies>true</checkDependencies>
+          <newVersion>${project.version}</newVersion>
+          <failOnUnresolvedArtifacts>true</failOnUnresolvedArtifacts>
           <analysisConfiguration>
-            <revapi.semver.ignore>
+            <revapi.reporter.json>
+              <minSeverity>NON_BREAKING</minSeverity>
+              <minCriticality>documented</minCriticality>
+              <output>${project.build.outputDirectory}/revapi-result.json</output>
```

Below target:
```suggestion
              <output>${build.directory}/revapi-result.json</output>
```

Otherwise the file in the classes folder.